nodejs/eks/examples/migrate-nodegroups/nginx-ing-cntlr-rbac.ts

```diff
@@ -0,0 +1,180 @@
+import * as k8s from "@pulumi/kubernetes";
+import * as pulumi from "@pulumi/pulumi";
+
+// Create a ServiceAccount.
+export function makeNginxServiceAccount(
+    name: string,
+    provider: k8s.Provider,
+    namespace: pulumi.Input<string>,
+): k8s.core.v1.ServiceAccount {
+    return new k8s.core.v1.ServiceAccount(
+        name,
+        {
+            metadata: {
+                namespace: namespace,
+            },
+        },
+        {
+            provider: provider,
+        },
+    );
+}
+
+// Create a ClusterRole.
+export function makeNginxClusterRole(
+    name: string,
+    provider: k8s.Provider,
+): k8s.rbac.v1.ClusterRole {
+    return new k8s.rbac.v1.ClusterRole(
+        name,
+        {
+            rules: [
+                {
+                    apiGroups: [""],
+                    resources: ["configmaps", "endpoints", "nodes", "pods", "secrets"],
+                    verbs: ["list", "watch"],
+                },
+                {
+                    apiGroups: [""],
+                    resources: ["nodes"],
+                    verbs: ["get"],
+                },
+                {
+                    apiGroups: [""],
+                    resources: ["services"],
+                    verbs: ["get", "list", "watch"],
+                },
+                {
+                    apiGroups: ["extensions"],
```

Comment:
Hmm, I guess you might have to update RBAC if you did change the apiVersion of the Ingress.